consensus/polybft/fsm.go

```diff
@@ -203,24 +203,6 @@ func (f *fsm) stateTransactions() []*types.Transaction {
 		}
 	}
 
-	if f.isEndOfSprint {
-		for _, bundle := range f.bundleProofs {
-			inputData, err := bundle.EncodeAbi()
```

Comment:
It seems we don't need `EncodeAbi` (possibly even `DecodeAbi`, needs checking) anymore? Therefore we don't need to make `BundleProof` implement `StateTransactionInput` interface anymore...?

Reply:
Starting just after this PR is merged (we need this for the release), I'll start removing everything related to bundles, I don't want to introduce more that just the necessary in this PR, makes sense?